java-frontend/src/main/java/org/sonar/java/model/declaration/MethodTreeImpl.java

```diff
@@ -269,9 +263,9 @@ public Boolean isOverriding() {
     if (symbol == null) {
       return null;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Null is returned but a "Boolean" is expected. ([squid:S2447](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2447))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.java%3Ajava&branch=SONARJAVA-1927&issues=AWDq5lGyo_TTbi26M3Q9&open=AWDq5lGyo_TTbi26M3Q9)

[ISSUE_KEY]: AWDq5lGyo_TTbi26M3Q9

Reply:
Looks like a FP to me. Method is annotated with `@Nullable`.

Reply:
https://jira.sonarsource.com/browse/SONARJAVA-2616

Reply:
This issue has been resolved by a user in SonarQube

[ISSUE_STATUS]: RESOLVED